### ISSUE TITLE
Grunt.option documentaion is incorrect about boolean arguments

### ISSUE THREAD

**Issue by mwisnicki** [gruntjs/grunt#1252]
http://gruntjs.com/api/grunt.option still claims that one can use shorter boolean options but this is not the case.

See also #603.

**Comment by cspotcode** [comment]
Yeah, @FremyCompany's right... this bug's been documented for 2 years, and still no 0.5.0.

In the meantime I've written a PR to fix the documentation: gruntjs/grunt-docs#136
